src/frontend/Cargo.toml

```diff
@@ -32,6 +32,7 @@ itertools = "0.10"
 meta-client = { path = "../meta-client" }
 moka = { version = "0.9", features = ["future"] }
 openmetrics-parser = "0.4"
+partition = { path = "../partition" }
```

Comment:
I think we should always use `region` instead of `partition`.

Reply:
"partition" here is a verb rather than the synonym of region. This crate involves managing the rules to partition region across datanodes.

Reply:
I think "region" is more appropriate as a new crate's name. "partition" seems too narrow for representing the crate's main functionality, that the crate is mainly used to **_produce the regions frontend uses_**. Besides, "partition" is excessively used all around DataFusion, I think it's better to make a little difference.

Reply:
If using `partition` as a verb, I think it is reasonable.